tests/frac-sym/input.dat

```diff
@@ -0,0 +1,16 @@
+molecule {
+0 2
+F
+}
+
+set frac_occ [ +3, +4, +5, -3, -4, -5 ]
+set frac_val [ 1.0, 1.0, 1.0, 6.666666666666666e-01, 6.666666666666666e-01, 6.666666666666666e-01 ]
+set frac_start 1
+set scf_type direct
+set df_scf_guess false
+set reference uhf
+set basis sto-6g
+
+computed_energy = energy('scf')
+
+compare_values(-98.6343474374526465, computed_energy, 8, "Fractional occupation energy")  #TEST
```

Comment:
possibly checking energy to greater precision than computed

Reply:
Good catch. Do you suggest increasing convergence or checking this less tightly? To tell the difference between frac and non-frac, we don't need all 8 decimals, but it might be a good as a "did anything break" check.

Reply:
In that case, I'd compute the ref with conv=10, then set conv=8 in the test and check to 8.

(If check to 6 had been ok, it's still nice to compute the ref with >=8.)